pkg/windowsdesktop/bundle/Microsoft.WindowsDesktop.App.Bundle.bundleproj

```diff
@@ -34,9 +34,9 @@
       <NETCoreAppInstallerMsiRID Include="@(InsertionPackageRID)" Name="host" InsertionName="SharedHost" />
 
       <NETCoreAppInstallerMsiRID
-        Id="VS.Redist.Common.NetCore.%(InsertionName).%(Arch).7.0"
-        Version="$(VSRedistCommonNetCoreSharedFrameworkx6470PackageVersion)"
-        MsiFileName="dotnet-%(Name)-$(MicrosoftNETCoreAppRuntimewinx64Version)-%(Identity).msi" />
+        Id="VS.Redist.Common.NetCore.%(InsertionName).%(Arch).8.0"
+        Version="$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)"
+        MsiFileName="dotnet-%(Name)-$(VSRedistCommonNetCoreSharedFrameworkx6480PackageVersion)-%(Identity).msi" />
```

Comment:
I'm a bit unsure about this tbh... but I think we can iterate on it later

Reply:
Second bit should be `MicrosoftNETCoreAppRuntimewinx64Version`. Otherwise you'll always get a non-stable MSI file name.